src/Components/calculator.js

```diff
@@ -1,35 +1,47 @@
 import './calculator.css';
-import Input from './input';
+import { useState } from 'react';
+import calculators from '../Logic/calculators';
 
 const Calculator = () => {
-  const value = 0;
+  const [result, setResult] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const eventHandler = (val) => {
+    const Result = calculators(result, val.target.value);
+    setResult(Result);
+  };
+
+  const displayValue = result.next || result.total || '0';
   return (
     <div className="main-calculator">
       <div className="row">
-        <Input value={value} />
+        <input className="header" type="number" value={displayValue} />
         <section className="calculator-wrapper">
-          <button type="button">AC</button>
-          <button type="button" className="plus-moins">+/-</button>
-          <button type="button" className="division">%</button>
-          <button type="button" className="color">&#247;</button>
-          <button type="button">7</button>
-          <button type="button">8</button>
-          <button type="button">9</button>
-          <button type="button" className="color">x</button>
+          <button type="button" onClick={eventHandler} value="AC">AC</button>
+          <button type="button" onClick={eventHandler} value="+/-" className="plus-moins">+/-</button>
+          <button type="button" onClick={eventHandler} value="%" className="division">%</button>
+          <button type="button" onClick={eventHandler} value="÷" className="color">&#247;</button>
+          <button type="button" onClick={eventHandler} value="7">7</button>
+          <button type="button" onClick={eventHandler} value="8">8</button>
+          <button type="button" onClick={eventHandler} value="9">9</button>
+          <button type="button" onClick={eventHandler} value="x" className="color">&#215;</button>
```

Comment:
- [ ] It would be better if we pass `props` to the child component from our `calculator.js`(parent) component. ***Hint***: We can create a `Button` component and pass the props from the parent(Calculator) component to the child(Button) component